lib/Yam/Agama/Pom/GrubMenuBaseBug1231658Page.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles common grub screen actions, workaround for bsc#1231658.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Yam::Agama::Pom::GrubMenuBaseBug1231658Page;
+use strict;
+use warnings;
+use testapi;
+
+use Mojo::Util 'trim';
+
+sub new {
+    my ($class, $args) = @_;
+    return bless {
+        key_edit_entry => 'e'
+    }, $class;
+}
+
+sub edit_current_entry {
+    my ($self) = @_;
+    wait_screen_change { send_key($self->{key_edit_entry}) };
+}
+
+sub select_first_entry {
+    my $grub_menu = $testapi::distri->get_grub_menu_agama();
+    my $grub_entry_edition = $testapi::distri->get_grub_entry_edition();
+    my @params = split ' ', trim('console=ttyAMA0 console=tty', '');
```

Comment:
you don't need the second parameter.

Reply:
Updated, and verified https://openqa.opensuse.org/tests/4582077#

Reply:
please, remove the trim and its import if it is not needed, just create a hash ref with the params.

Reply:
Updated, VR https://openqa.opensuse.org/tests/4582236#